app/assets/javascripts/users/show.js.coffee

```diff
@@ -11,8 +11,7 @@ IS.onReady "users/show", ->
   nav_list = []
 
   ($ '#user_filter li').each (index) ->
-    if index != 0
-      nav_list.push ($ @).text()
```

Comment:
This was the culprit.  Someone *explictly* chose to not include the Liked Projects tab (index 0) in the nav_list array. :-1: